assets/src/adaptivity/capi.ts

```diff
@@ -98,12 +99,70 @@ export interface ICapiVariableOptions {
   writeonly?: boolean;
   allowedValues?: string[];
   bindTo?: string;
+  shouldConvertNumbers?: boolean;
 }
 
 export interface JanusConditionProperties extends ConditionProperties {
   id: string;
   type?: CapiVariableTypes;
 }
+export const parseArrayWithStringConversion = (val: unknown): unknown[] => {
```

Comment:
also this one maybe should go into the utils/common and have a unit test to show that it isn't altering the arrays